src/pages/TestBallotDeckScreen.tsx

```diff
@@ -58,10 +58,7 @@ const generateTestDeckBallots = ({
             ? 2
             : c.type === 'candidate'
             ? (c as CandidateContest).candidates.length
-            : /* istanbul ignore next - TODO: Tally: Remove this line once tally code is completed */
-            c.type === 'ms-either-neither'
-            ? 9 // TODO: Tally: 9 because: no vote + A + B
-            : 0
+            : 2 // ms-either-neither
```

Comment:
Else case as 'ms-either-neither` worries me. I feel like this is gonna come back to bite us if it's not an explicit contest type check.

Reply:
Yeah, I did this because the other approach then caused a code coverage problem, but I can `istanbul` ignore that last line. will change.